en/db-migrations.md

```diff
@@ -119,6 +119,33 @@ $migration::run([
 ]);
 ```
 
+## Migration methods
+
+Each migration is an separate class that works as entity for specific database table.
+Inside each class there are different methods that can occur during migration running.
+
+You can also execute whatever logic you want inside each migration, without any limitation.
+
+Here are tables with Migration Class methods. Table is sorted by order of execution: from earliest to latest.
```

Comment:
What do you mean by _here are tables...._?

Reply:
2 HTML tables below